test/marginal/resolver-paths.cpp

```diff
@@ -30,6 +30,7 @@ int main(){
     ret+=test_normalizer("/hello world/banana/ww/a", "/hello world/banana/", "quick-js/./.././ww/a", false, false);
     ret+=test_normalizer("/a/b../", "/a/", "b../", false, false);
 
-    assert(ret=0);
+    std::cerr<<"ret = "<<ret<<"\n";
+    assert(ret==0);
```

Comment:
This fixes where the test fails even when ret is eq to 0 ;)

Reply:
LOL in theory my IDE should have warned me, somehow I missed its message.